Drv/LinuxGpioDriver/LinuxGpioDriverComponentImpl.cpp

```diff
@@ -311,64 +324,63 @@ namespace Drv {
   void LinuxGpioDriverComponentImpl ::
     intTaskEntry(void * ptr) {
 
-      FW_ASSERT(ptr);
-      LinuxGpioDriverComponentImpl* compPtr = (LinuxGpioDriverComponentImpl*) ptr;
+    FW_ASSERT(ptr);
+    LinuxGpioDriverComponentImpl* compPtr = (LinuxGpioDriverComponentImpl*) ptr;
```

Comment:
Can you use a static_cast here.  (LinuxGpioDriverComponentImpl*) can be dangerous.

Reply:
Is all of this just formatting?  If so, don't change it.

Reply:
This is just formatting cause it was mixed tabs, 2 and 4 space indentation. I can revert this I guess but it was pretty bad.